data.sql

```diff
@@ -1,5 +1,7 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO vet_clinic (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
```

Comment:
Great job @IsmaelMastronardi 🤝

- [ ] As stated in the project requirements, kindly ensure you are inserting data into the `animals` table not the `vet_clinic`.